src/Mod/Draft/Draft.py

```diff
@@ -143,6 +143,7 @@
 from draftfunctions.heal import heal
 
 from draftfunctions.move import (move,
+                                 moveSubElements,
```

Comment:
Can you name any new command using `snake_case` style?

Reply:
Sure! That's just a draft